### PR TITLE
yeeeey

### PR DESCRIPTION
**Pull request by hiroho-t**
yey

### CODE REVIEW

**Comment by mark-drastrup** [comment]
@hirohot great job!!

File: index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <title>Git demo</title>
   </head>
   <body>
-    <h1>Welcome to the git demo</h1>
+    <h1>Welcome to the git demo=^_^=</h1>
```
Comment (a-yasutommy) [inline index.html]: @hirohot  How about this??? (@_@)

Comment (hiroho-t) [reply index.html]: lovelyyy

**Review by mark-drastrup — APPROVED** [review]

**Review by nozomi-I — APPROVED** [review]
ok

**Review by a-yasutommy — APPROVED** [review]
Thank you @hirohot !

**Pull request merged**